src/olympia/ratings/models.py

```diff
@@ -48,13 +48,13 @@ def update_ratings_and_addons_denormalized_fields(self, pairs):
 
     def delete(self):
         pairs = tuple(self.values_list('addon_id', 'user_id'))
-        rval = self.update(deleted=F('id'))
+        rval = self.order_by().update(deleted=F('id'))
```

Comment:
Note https://docs.djangoproject.com/en/4.2/ref/models/querysets/#ordered-queryset (not a current problem for us... but an interesting limitation)